integration/code/connecting_to_your_data/database/postgres.py

```diff
@@ -33,18 +34,37 @@
 """
 config = config.replace("<YOUR_CONNECTION_STRING_HERE>", CONNECTION_STRING)
 
-sanitize_yaml_and_save_datasource(context, config, overwrite_existing=True)
+print(f"Adding Datasource")
```

Comment:
I'm guessing the print statements were left in inadvertently for debugging. I don't think they add to the UX